model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -225,7 +225,17 @@ class LicenseInfoResolver(
                 }
             }
 
-            unmatchedCopyrights.getOrPut(findings.provenance) { mutableSetOf() } += matchResult.unmatchedCopyrights
+            unmatchedCopyrights.getOrPut(findings.provenance) {
+                mutableSetOf()
+            } += matchResult.unmatchedCopyrights.map { copyrightFinding ->
+                ResolvedCopyrightFinding(
```

Comment:
Can we somehow reuse `LicenseInfoResolver.resolveCopyrights()` by making that function public and moving it out of the class context?

Reply:
Good catch. It draw me to `relativeFindingsPath` which I had overlooked before.